sidebars.ts

```diff
@@ -120,6 +120,7 @@ const sidebars: SidebarsConfig = {
         'room/ownership-and-lifetime-flags',
         'room/room-server-options',
         'room/offline-mode',
+        'room/realtimemodel-primitives',
```

Comment:
I think this should be a sub page under `room/realtimemodel` imo. In which case we should update the sidebar entry to have a child under realtime model and also make a folder in `room` called `realtimemodel` so the paths match